clusterdock/topologies/cdh/actions.py

```diff
@@ -168,8 +207,38 @@ def start(args):
                         getfqdn(), hue_server_host_port)
             break
 
-    logger.info("Deploying client configuration...")
-    deployment.cluster.deploy_client_config().wait()
+    realm = cluster.network_name.upper()
+    if args.kerberos:
+        logger.info('Updating config for kerberos...')
+        kerberos_config = {'SECURITY_REALM': realm,
+                           'KDC_HOST': KDC_HOST_NAME,
+                           'KRB_MANAGE_KRB5_CONF': True,
+                           'KRB_ENC_TYPES': 'aes256-cts-hmac-sha1-96'}
+        deployment.cm.update_config(kerberos_config)
+
+        logger.info('Importing kerberos admin credentials...')
+        deployment.cm.import_admin_credentials('{}@{}'.format(CLDR_MGR_PRINCIPAL_USER, realm),
+                                               CLDR_MGR_PRINCIPAL_PW).wait()
+
+        logger.info('Configuring for kerberos...')
+        config_kerberos_cmd = deployment.cluster.configure_for_kerberos()
```

Comment:
Is there a reason we need to create a local var with this command? Why not just roll it up as 
```
if not deployment.cluster.configure_for_kerberos().wait().success:
```

Same comment for other `..._cmd`s below